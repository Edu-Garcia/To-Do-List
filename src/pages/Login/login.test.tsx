import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom';
import { Login } from '.'
import '@testing-library/jest-dom';

describe('Login Tests', () => {
  it('snapshot test login page', () => {
    const { container } = render(
      <BrowserRouter>
        <Login />
      </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
  });
});