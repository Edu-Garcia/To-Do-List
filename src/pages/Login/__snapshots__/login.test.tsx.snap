// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Tests snapshot test login page 1`] = `
<div>
  <div
    class="container"
  >
    <form>
      <h1>
        Login
      </h1>
      <div
        class="form-group"
      >
        <div
          class="input-container"
        >
          <label
            for="email"
          >
            E-mail
          </label>
          <input
            id="email"
            name="email"
            placeholder="Informe seu e-mail"
            required=""
            type="email"
            value=""
          />
        </div>
        <div
          class="input-container"
        >
          <label
            for="password"
          >
            Senha
          </label>
          <input
            id="password"
            name="password"
            placeholder="Informe sua senha"
            type="password"
            value=""
          />
        </div>
        <button
          class="button undefined"
          type="submit"
        >
          Entrar
        </button>
      </div>
      <p
        class="registerText"
      >
        NÃ£o possui conta? 
        <a
          href="/register"
        >
          Cadastro
        </a>
      </p>
    </form>
  </div>
</div>
`;
